lib/standard_deviation.rb

```diff
@@ -1,5 +1,26 @@
+require 'pry'
+
 ages = [24, 30, 18, 20, 41]
 
-# Your code here for calculating the standard deviation
+def std_dev(ages)
```

Comment:
This code should be inside the Event class. Generally, we don't want to have methods that aren't wrapped in a class or module